tests/debuginfo/constant-ordering-prologue.rs

```diff
@@ -1,6 +1,7 @@
 //@ min-lldb-version: 310
 
 //@ ignore-aarch64-unknown-ferrocenecoretest
+//@ ignore-aarch64-darwin-ferrocenecoretest
```

Comment:
This target does not exist.

Reply:
Removed both directives. Added them to `ferrocene/ignored-tests.toml`.